keyboards/ergodox_ez/keymaps/mac_win_german_programmer/.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
This file can be removed, the top-level gitignore now has the correct case to ignore these :)